### PR TITLE
Solarian Warlord Armor

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
Custom armor for all Solarian warlords. Except the SFA, because it's dead. WIP, need to iron out one last thing. Also edits the Vulture description

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!wip

**Comment by TheDanseMacabre** [comment]
!review

File: code/modules/clothing/spacesuits/void/misc.dm [hunk 1]
```diff
@@ -786,3 +786,108 @@
 	)
 	contained_sprite = TRUE
 	species_restricted = list(BODYTYPE_TAJARA)
+
+/obj/item/clothing/head/helmet/space/void/sol/srf
+	name = "solarian restoration front voidsuit helmet"
+	desc = "An uparmored variant of the gargoyle voidsuit helmet, with a solid visor and redundant comms antenna. Intended for urban combat operations."
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/void/misc.dm:792]:
```suggestion
	desc = "An uparmoured variant of the gargoyle voidsuit helmet, with a solid visor and redundant comms antenna. Intended for urban combat operations."
```

File: code/modules/clothing/spacesuits/void/misc.dm [hunk 2]
```diff
@@ -786,3 +786,108 @@
...
+	desc = "A variant of the Solarian Armed Forces Type-4 \"Gargoyle\" voidsuit, the Type-5 \"Cyclops\" is the urban combat version of the Gargoyle. While practical testing showed it was a marginal improvement over the Type-4 at best, the SRF have taken to using this modification \
+	both on account of its intimidating design and in an effort to distinguish its forces from others using the Type-4, warlord or not. This one has the SRF's flag on the breastplate."
+	icon = 'icons/obj/contained_items/voidsuits/warlordsuit.dmi'
+	icon_state = "srf_suit"
+	item_state = "srf_suit"
+	contained_sprite = TRUE
+
+/obj/item/clothing/head/helmet/space/void/sol/league
+	name = "anti-corporate league voidsuit helmet"
+	desc = "A sleek and waspish composite-armored voidsuit helmet, issued to the personnel of the Sol Alliance's military. This one has been modified by the forces of the League of Independent Corporate-Free Systems to match the colors of the Xanusian \"Vulture\" voidsuit, and also features a modified comms antenna."
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/void/misc.dm:809]:
```suggestion
	desc = "A sleek and waspish composite-armoured voidsuit helmet, issued to the personnel of the Sol Alliance's military. This one has been modified by the forces of the League of Independent Corporate-Free Systems to match the colors of the Xanusian \"Vulture\" voidsuit and also features a modified comms antenna."
```

File: code/modules/clothing/spacesuits/void/misc.dm [hunk 3]
```diff
@@ -786,3 +786,108 @@
...
+	icon = 'icons/obj/contained_items/voidsuits/warlordsuit.dmi'
+	icon_state = "league_helmet"
+	item_state = "league_helmet"
+	contained_sprite = TRUE
+
+/obj/item/clothing/suit/space/void/sol/league
+	name = "anti-corporate league voidsuit"
+	desc = "A midweight Zavodskoi-manufactured voidsuit designed for the Solarian Armed Forces, the Type-4 \"Gargoyle\" is the primary armored voidsuit in use by the Alliance military. This one has been modified by the forces of the League of Independent Corporate-Free Systems to match the colors of the Xanusian \
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/void/misc.dm:817]:
```suggestion
	desc = "A midweight Zavodskoi-manufactured voidsuit designed for the Solarian Armed Forces, the Type-4 \"Gargoyle\" is the primary armoured voidsuit in use by the Alliance military. This one has been modified by the forces of the League of Independent Corporate-Free Systems to match the colors of the Xanusian \
```

File: code/modules/clothing/spacesuits/void/misc.dm [hunk 4]
```diff
@@ -786,3 +786,108 @@
...
+	\"Vulture\" voidsuit, to make it more easily identified by friendly Coalition forces."
+	icon = 'icons/obj/contained_items/voidsuits/warlordsuit.dmi'
+	icon_state = "league_suit"
+	item_state = "league_suit"
+	contained_sprite = TRUE
+
+/obj/item/clothing/head/helmet/space/void/sol/fsf
+	name = "free solarian fleets voidsuit helmet"
+	desc = "A sleek and waspish composite-armored voidsuit helmet, issued to the personnel of the Sol Alliance's military. This one has been modified by the forces of the Free Solarian Fleets to make it more easily identifiable from other forces using the Type-4, warlord or not."
```
Comment (SleepyGemmy) [inline code/modules/clothing/spacesuits/void/misc.dm:826]:
```suggestion
	desc = "A sleek and waspish composite-armoured voidsuit helmet, issued to the personnel of the Sol Alliance's military. This one has been modified by the forces of the Free Solarian Fleets to make it more easily identifiable from other forces using the Type-4, warlord or not."
```

**Review by SleepyGemmy — APPROVED** [review]
sprites look good.
code looks good, other than some minor grammatical nitpicks.

File: code/modules/clothing/spacesuits/void/misc.dm (hunk 2 above)
Comment (TheDanseMacabre) [reply code/modules/clothing/spacesuits/void/misc.dm:809]: I am not British.

File: code/modules/clothing/spacesuits/void/misc.dm (hunk 4 above)
Comment (TheDanseMacabre) [reply code/modules/clothing/spacesuits/void/misc.dm:826]: I am not British.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**